docs/source/services/timelock_service/installation.rst

```diff
@@ -94,17 +93,14 @@ Add client(s) to Timelock
           sslConfiguration:
             trustStorePath: var/security/truststore.jks
 
-10. Configure Timelock to respond to your clients.
-    Add each client as a named entry in the ``clients`` block of your :ref:`timelock-server-configuration`.
+10. (Optional) For verification purposes, you may retrieve a timestamp from each client you are configuring to use TimeLock.
```

Comment:
if this is optional, then shouldn't 15 also be optional?

Reply:
Correct, 15 is optional. I've added that.